src/components/filmDescription/__snapshots__/filmDescription.test.js.snap

```diff
@@ -1,34 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`FilmDescription renders correctly 1`] = `
-<div
-  className="film-description clearfix"
->
-  <div
-    className="film-description__image-wrapper"
-  >
-    <img
-      className="film-description__image"
-    />
-  </div>
-  <div
-    className="film-description__info-wrapper"
-  >
-    <h2
-      className="film-description__header"
-    />
-    <strong
-      className="film-description__useful-info display-block"
-    />
-    <span>
-      Year: NaN 
-    </span>
-    <span>
-      undefined min
-    </span>
-    <p
-      className="film-description__narration"
-    />
-  </div>
-</div>
+<ContextConsumer>
+  <Component />
```

Comment:
Inside test try to import not connected component, to have more descriptive snapshot.
like this:
import { FilmDescription } from './filmDescription'

Reply:
Didn't get this point...